content/en/docs/creating-addons/testing/install-specific-version-of-addon.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Installing a specific version of an add-on in a staging environment
+linkTitle: Installing a specific version of an add-on in a staging environment
+---
+
+Add-on services are typically installed using the OpenShift Cluster Manager web console, by selecting the specific add-on from the **Add-ons** tab and clicking **Install**.
+However, only the latest version of an add-on service can be installed using the OpenShift Cluster Manager console.
+
+In some cases, you might need to install an older version of an add-on, for example, to test the upgrade of an add-on from one version to the next.
+Follow this procedure to install a specific version of an add-on service in a staging environment.
+
+**IMPORTANT:**
+Installing an add-on service using this procedure is only recommended for testing upgrades in a staging environment and is not supported for customer-facing production workloads.
+
+## Prerequisites
+
+* You have the `version_select` capability added to your organization by creating a merge request to the [_ocm-resources respository_](https://gitlab.cee.redhat.com/service/ocm-resources).
+
+    For more information about enabling the `version_select` capability, see [_organization YAML example_](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-stage/orgs/15081438.yaml#L97) and [_merge request example_](https://gitlab.cee.redhat.com/service/ocm-resources/-/merge_requests/3271).
```

Comment:
Pls use the commit hash instead of "master", so the link doesn't get outdated:

https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/35198592/data/uhc-stage/orgs/15081438.yaml#L97